lib/Client.php

```diff
@@ -24,20 +24,21 @@
 namespace OCA\SharePoint;
 
 use Exception;
+use OCP\ILogger;
```

Comment:
It seems ILogger was not used before and I don’t see it used in the diff?

Reply:
In `ensureConnection()`, but let me replace that as well.